modulemd/katello_modulemd.yaml

```diff
@@ -0,0 +1,19 @@
+---
+document: modulemd
+version: 2
+data:
+    name: katello
+    stream: latest
+    summary: Katello module
+    description: >-
+        A module to make Katello easier to install.
+    license:
+        module:
+            - GPLv3
+    dependencies:
```

Comment:
Should it also depend on the Foreman module?

Reply:
That's a good call.

Reply:
And nodejs is also a build requirement

Reply:
While it is a build requirement, I don't think *users* need this, so I'd keep the number of modules as short as possible.

Reply:
But you literally have a `buildrequires` section. IMHO it would be nice to be explicit about what we do.

Reply:
indeed. that slightly slipped my mind.